routing/control_tower.go

```diff
@@ -266,6 +279,8 @@ func (p *controlTower) SubscribePayment(paymentHash lntypes.Hash) (
 // with the current state of every inflight payment is always sent out
 // immediately.
 func (p *controlTower) SubscribeAllPayments() (*ControlTowerSubscriber, error) {
+	p.newSubscriptionMtx.Lock()
+	defer p.newSubscriptionMtx.Unlock()
```

Comment:
nit: add empty line after the locks to visually separate them.

Reply:
Removed locks